README.md

```diff
@@ -82,7 +85,12 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+<adjective> ::= "smelly" | "lazy"
+<noun> ::= "dog" | "cat"
+<verb> ::= "ate" | "ran"
+<adverb> ::= "slowly" | "noisily"
+
+sentence ::= "the" <adjective> <noun> <verb> <adverb>
```

Comment:
This grammar forces you to have exactly one adjective and one adverb. The original allows zero or more adjectives and zero or one adverb.

I was looking for something like

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
  <noun> ::= "dog" | "cat"
  <verb> ::= "ate" | "ran"
  <adverb> ::= "slowly" | "noisily" | ""
~~~

# -4